src/marshmallow_sqlalchemy/convert.py

```diff
@@ -313,17 +313,22 @@ def _add_column_kwargs(self, kwargs, column):
         """Add keyword arguments to kwargs (in-place) based on the passed in
         `Column <sqlalchemy.schema.Column>`.
         """
-        if column.nullable:
-            kwargs["allow_none"] = True
-        kwargs["required"] = not column.nullable and not _has_default(column)
+        if hasattr(column, "nullable"):
+            if column.nullable:
+                kwargs["allow_none"] = True
+            kwargs["required"] = not column.nullable and not _has_default(column)
+        # If there is no nullable attribute, we are dealing with a property
+        # that does not derive from the Column class. Mark as dump_only.
```

Comment:
@sloria do you agree with branching based on the `nullable` property, or do you think a specific `isinstance` check would be better?

Reply:
I like this approach 👍